new-client/src/plugins/GeosuiteExport/GeosuiteExportModel.js

```diff
@@ -6,6 +6,9 @@ import { Vector as VectorLayer } from "ol/layer";
 import { intersects, within } from "ol/format/filter";
 import { hfetch } from "utils/FetchWrapper";
 import { WFS } from "ol/format";
+import { saveAs } from "file-saver";
+
+const JSZip = require("jszip");
```

Comment:
I would have updated this one for you, but i didn't manage to get the export working so i decided it's better if you do it. Anyways, let's use ES6 imports. Changing `const JSZip = require("jszip");` to `import JSZip from "jszip"` should work according to their GitHub.